src/components/NavigationSection.svelte

```diff
@@ -1,18 +1,20 @@
 <li class='coz-nav-section' on:click='dispatch(event)'>
   <a on:click='toggle()' aria-controls='{{`coz-nav-pop--${hash}`}}' aria-busy='{{busy}}' data-icon='{{icon}}'>
-    {{t(label)}}
+    {{label}}
   </a>
-  {{#if items}}
+  {{#if items && items.length}}
   <div class='{{`coz-nav-pop coz-nav-pop--${hash}`}}' id='{{`coz-nav-pop--${hash}`}}' aria-hidden={{closed}}>
-    {{#if items.error}}
+  {{#if items[0].error}}
```

Comment:
Maybe we can prefer `#if items.length && items[0].error` here to prevent some errors. What do you think?